src/garage/tf/models/gaussian_cnn_model.py

```diff
@@ -86,8 +90,10 @@ class GaussianCNNModel(Model):
                exponential transformation
             - softplus: the std will be computed as log(1+exp(x))
         layer_normalization (bool): Bool for using layer normalization or not.
+
     """
 
+    # pylint: disable=dangerous-default-value
```

Comment:
please don't disable this -- it's a real problem.

which value is dangerous?

Reply:
just change the lists to tuple...

Reply:
I forgot to remove thesde